app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenAnimatedActionCard.kt

```diff
@@ -0,0 +1,43 @@
+package com.x8bit.bitwarden.ui.platform.components.card
+
+import androidx.compose.animation.animateContentSize
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+/**
+ * A version of [BitwardenActionCard] which will animate the height of the card
+ * based on the [isVisible] parameter.
+ */
+@Composable
+fun BitwardenAnimatedActionCard(
+    isVisible: Boolean,
+    cardTitle: String,
+    actionText: String,
+    onActionClick: () -> Unit,
+    onDismissClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    leadingContent: @Composable (() -> Unit)? = null,
+    ) {
+    Box(
```

Comment:
Any reason not to user `AnimatedVisibility`?

Reply:
smoothness of the transition of the content below moving up, AV was my original play https://livefront.slack.com/archives/C05LE6KHAHM/p1727796222300119